packages/editor/src/store/selectors.js

```diff
@@ -519,10 +549,9 @@ export function isEditedPostAutosaveable( state ) {
 		return true;
 	}
 
-	// If the title, excerpt or content has changed, the post is autosaveable.
-	const autosave = getAutosave( state );
+	// If the title or excerpt has changed, the post is autosaveable.
 	return [ 'title', 'excerpt' ].some( ( field ) => (
```

Comment:
Should we use `AUTOSAVE_PROPERTIES` here?

Reply:
`AUTOSAVE_PROPERTIES` also has the `content` property, which isn't used on that line, it's compared in the preceding lines in a slightly different fashion. I suppose something like `omit( AUTOSAVE_PROPERTIES, 'content' )` could be used. What do you think?

Reply:
Eh, leave it as is 🙂